src/fusor/models.py

```diff
@@ -285,27 +290,6 @@ class TemplatedSequenceElement(BaseStructuralElement):
     )
 
 
-class GeneElement(BaseStructuralElement):
-    """Define Gene Element class."""
-
-    type: Literal[FUSORTypes.GENE_ELEMENT] = FUSORTypes.GENE_ELEMENT
-    gene_descriptor: GeneDescriptor
```

Comment:
similar to above, rather than deleting this class I think we can just change old line 292 to `gene: Gene`